content/vault/v1.20.x/content/docs/updates/important-changes.mdx

```diff
@@ -348,7 +348,24 @@ then performs a second password update.
 While users still receive the latest password, the secondary update may lead to
 unexpected LDAP activity and cause confusion interpreting audit logs.
 
-### Recommendation
+#### Recommendation
 
 Send check-in requests directly to the active node of the primary cluster to
 prevent duplicate password rotations on the LDAP server.
+
+### Missed events with multiple event clients ((#missed-events)) <EnterpriseAlert inline="true" />
+
+| Change       | Affected version       | Fixed version
+| ------------ | ---------------------- | --------------------
+| Known issue  | 1.20.x+ent, 1.19.x+ent | None
+
+Users may miss events if two or more event clients subscribe to the same
+performance standby node in a cluster, and each client specifies the same
+namespace and event type filters. If one client disconnects, the other(s) will
+no longer receive events.
```

Comment:
```suggestion
Users may miss events when multiple clients subscribe to the same performance
standby node in a cluster with the same namespace and event type filters because
one client disconnecting unsubscribes the remaining clients who no longer
receive events.
```
Edited to match 1.19.x language